### PR TITLE
hakrawler: init at 20201224-e39a514

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Add hakrawler (https://github.com/hakluke/hakrawler) a Web crawler for the discovery of endpoints and assets.

Related to #81418

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
Please change the commit message to ``hakrawler: init at 20201224-e39a514``.

File: pkgs/tools/security/hakrawler/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  src = fetchFromGitHub {
+    owner = "hakluke";
+    repo = "hakrawler";
+    inherit rev;
+    sha256 = "0wpqfbpgnr94q5n7i4zh806k8n0phyg0ncnz43hqh4bbdh7l1y8a";
+  };
+
+  vendorSha256 = "18zs2l77ds0a3wxfqcd91h269g0agnwhginrx3j6gj30dbfls8a1";
+
+  doCheck = true;
```
Comment (SuperSandro2000) [inline pkgs/tools/security/hakrawler/default.nix:20]:
```suggestion
```

This is true by default.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107616` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>hakrawler</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107616` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>hakrawler</li>
  </ul>
</details>

File: pkgs/tools/security/hakrawler/default.nix (hunk 1 above)
Comment (fabaff) [reply pkgs/tools/security/hakrawler/default.nix:20]: Thanks for your feedback.

Removed

Comment (SuperSandro2000) [reply pkgs/tools/security/hakrawler/default.nix:20]: I think you did something wrong when force pushing because the line is back.

Comment (fabaff) [reply pkgs/tools/security/hakrawler/default.nix]: Yes, it did. Not saving the file and I can force push all day long with no effect :wink:

File: pkgs/tools/security/hakrawler/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  meta = with stdenv.lib; {
+    description = "Web crawler for the discovery of endpoints and assets";
+    homepage = "https://github.com/hakluke/hakrawler";
+    longDescription =  ''
+      Simple, fast web crawler designed for easy, quick discovery of endpoints
+      and assets within a web application.
+    '';
+    license = licenses.mit;
+    maintainers = with maintainers; [ fab ];
+    platforms = platforms.linux ++ platforms.darwin;
```
Comment (SuperSandro2000) [inline pkgs/tools/security/hakrawler/default.nix:29]:
```suggestion
```

Default in buildGoModule

**Review by SuperSandro2000 — APPROVED** [review]

File: pkgs/tools/security/hakrawler/default.nix (hunk 2 above)
Comment (fabaff) [reply pkgs/tools/security/hakrawler/default.nix]: Removed, thanks.

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Pull request merged**